pkg/report/predicate/vuln.go

```diff
@@ -0,0 +1,89 @@
+package predicate
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"time"
+
+	"github.com/package-url/packageurl-go"
+	"golang.org/x/xerrors"
+
+	"github.com/aquasecurity/trivy/pkg/clock"
+	"github.com/aquasecurity/trivy/pkg/types"
+)
+
+// CosignVulnPredicate represents the Cosign Vulnerability predicate.
+// Cosign provides the CosignVulnPredicate structure in their repository.
+// But the type of Scanner.Result is defined as map[string]interface{}, which is difficult to use,
+// so we define our own.
```

Comment:
Add this link, please.
https://github.com/sigstore/cosign/pull/2096

Reply:
Alright. I've added the link.